### PR TITLE
Update seeds.rb

### PR DESCRIPTION
**Pull request by OlehSliusar**
We don't build a web server anymore

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -698,7 +698,7 @@ def require_updates?(real_attrs, seed_attrs)
...
+  description: "In this course you will learn Ruby, the language designed specifically with programmer happiness in mind. By the end of it all, you'll have built some pretty sweet projects including Tic Tac Toe, Hangman, and even Chess. You'll be able to put save and open files, test out your code, separate your spaghetti code into nice modular classes, and even reproduce some basic algorithms and data structures for solving complex problems. Basically, you're going to start feeling a whole lot more like a real programmer and that feeling will be justified.",
   position: course_position,
   you_learn: ["How to write clean, effective, modular code","Working with files and scripts", "Basic algorithms and data structures"],
   you_build: ["TicTacToe, Mastermind, and Hangman", "A command line server and browser","Chess (seriously)"],
```
Comment (arun1595) [inline db/seeds.rb]: @OlehSliusar This line has to be updated too

**Comment by KevinMulhern** [comment]
Derp, I forgot about this @OlehSliusar, thanks for creating this fix 😄 do you want to add Aruns suggestion to this PR?

**Comment by OlehSliusar** [comment]
@KevinMulhern yeah, I added the suggestion

**Comment by KevinMulhern** [comment]
LGTM 👍

**Pull request merged**